src/routes/version.rs

```diff
@@ -0,0 +1,52 @@
+use std::env;
+
+use axum::{
+    response::{IntoResponse, Response},
+    routing::get,
+    Json, Router,
+};
+use serde::Serialize;
+
+pub fn route() -> Router {
+    Router::new()
+        .route("/version", get(version))
+        .route("/version-details", get(version_details))
+}
+
+#[derive(Serialize)]
+struct VersionDetails {
+    version: &'static str,
+    version_bits: VersionBits,
+    git_commit_hash: &'static str,
+    current_endpoint_prefix: &'static str,
+    repository: &'static str,
+}
+
+#[derive(Serialize)]
+struct VersionBits {
+    major: &'static str,
+    minor: &'static str,
+    patch: &'static str,
+}
+
+async fn version() -> Response {
+    let v = env!("CARGO_PKG_VERSION");
+
+    return v.into_response();
+}
+
+async fn version_details() -> Response {
+    let vd = VersionDetails {
+        version: env!("CARGO_PKG_VERSION"),
+        version_bits: VersionBits {
+            major: env!("CARGO_PKG_VERSION_MAJOR"),
+            minor: env!("CARGO_PKG_VERSION_MINOR"),
+            patch: env!("CARGO_PKG_VERSION_PATCH"),
+        },
+        git_commit_hash: env!("GIT_COMMIT_HASH"),
+        current_endpoint_prefix: env!("CARGO_PKG_VERSION_MAJOR"),
+        repository: env!("CARGO_PKG_REPOSITORY"),
+    };
+
+    return Json(vd).into_response();
```

Comment:
```suggestion
    let version_details = VersionDetails {
        version: env!("CARGO_PKG_VERSION"),
        version_bits: VersionBits {
            major: env!("CARGO_PKG_VERSION_MAJOR"),
            minor: env!("CARGO_PKG_VERSION_MINOR"),
            patch: env!("CARGO_PKG_VERSION_PATCH"),
        },
        git_commit_hash: env!("GIT_COMMIT_HASH"),
        current_endpoint_prefix: env!("CARGO_PKG_VERSION_MAJOR"),
        repository: env!("CARGO_PKG_REPOSITORY"),
    };

    return Json(version_details).into_response();
```

Reply:
This is such a nit.

<img width="424" alt="image" src="https://github.com/user-attachments/assets/011ad6cd-9451-4ea5-931c-8e24df665776">

It's pretty clear what vd stands for – and it would not even be a variable if not for the fact that returning it like so was too cluttered visually. The variable name is shortened to reduce the declaration & return line lengths.

<img width="570" alt="image" src="https://github.com/user-attachments/assets/041ca141-46e9-4bec-a25b-d88ce2c69424">

Reply:
Using variable names consisting of one or two letters is a bad practice and we should apply the same standards to the entire code when possible. Here it might be clear what `vd` stands for, but similar naming might (and will) be detrimental elsewhere. Should you decide to remove the variable entirely, I will accept it.